helper/m4/configure.m4

```diff
@@ -8,14 +8,13 @@ AC_ARG_ENABLE([test-helper],
 AM_CONDITIONAL([test_helper], [test x$test_helper = xyes ])
 
 ##########################################################################
-# Enable/disable helper-ext
-# platform specific non portable extensions
+# Enable/disable Linux helpers
 ##########################################################################
-helper_linux=no
+helper_linux=yes
 AC_ARG_ENABLE([helper-linux],
-	[  --enable-helper-linux	build helper platform extensions (not portable)],
-	[if test "x$enableval" = "xyes"; then
-		helper_linux=yes
+	[  --enable-helper-linux   enable Linux helpers],
```

Comment:
As you are touching this line, could you please use AS_HELP_STRING here?

Reply:
@lumag I guess just copy paste code from odp to ofp :)

Reply:
Done in v3